snot.py

```diff
@@ -245,18 +232,27 @@ def options(self, parser, env=os.environ):
 
     def configure(self, options, conf):
         super(SlickAsSnotPlugin, self).configure(options, conf)
-        global config, testrun
         assert isinstance(conf, nose.config.Config)
+        self.options = options
+
+    def addSlickTestrun(self, testplan_name=None):
```

Comment:
Moved majority of configure step into it's own def that's called by prepareTests on line 331.